src/igvfd/types/source.py

```diff
@@ -0,0 +1,24 @@
+from snovault import (
+    collection,
+    load_schema,
+    calculated_property
+)
+from .base import (
+    Item,
+    paths_filtered_by_status,
```

Comment:
You are not using paths_filtered_by_status or ALLOWED or DELETED in your code below.  I tend to remove imports that I am not using.